### DISCUSSION TITLE
[visx/zoom] Zoom and center to specific node (Tree)

### DISCUSSION THREAD

**Discussion by czaplej** [airbnb/visx#1056] [Q&A]
According to this example [https://airbnb.io/visx/linktypes](https://airbnb.io/visx/linktypes) Is it possible to center screen and zoom to the specific node element??

**Comment by williaster** [comment]
Hey @czaplej  👋  thanks for checking out `visx`. Since this is a usage question and not an issue with `visx`, it might be more appropriate for our [github discussions](https://github.com/airbnb/visx/discussions) so I'm going to go ahead and move it over.

It'd also be helpful to provide more details about your expected behavior. For instance by `zoom` do you mean [`@visx/zoom`](https://airbnb.io/visx/zoom-i) like behavior? Or do you wish to filter nodes and animate things in and out? I think the best examples we have of animating are @techniq 's [here](https://codesandbox.io/s/github/techniq/vx-hierarchy-examples).

**Reply by czaplej** [reply]
I would check this example. Thanks a lot. 
![image](https://user-images.githubusercontent.com/29889157/107512390-0e62d280-6ba7-11eb-9a51-2b7a11a12a8d.png)
It will be very similar to this sample but also would be search input where u could find exactly node and after click scroll and zoom to the specific node
